src/components/alert/alert.test.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import Alert from ".";
+
+import Logger from "../../__internal__/utils/logger";
+
+// mock Logger.deprecate so that Typography (used for the alert dialog's heading) doesn't trigger a warning while running the tests
```

Comment:
comment: I don't think we need this. We should be careful about how we mock this so that we're not inadvertently hiding legitimate warnings.

Reply:
I'll try the tests without. It was already in the original enzyme test file so I figured we'd need it in the new test file too.